set_vcf_sample_id/README.md

```diff
@@ -0,0 +1,185 @@
+# Update a VCF header line sample ID
+
+This pipeline is useful when you have a set of single-sample VCFs
+in Cloud Storage in which the sample ID in the header needs to be changed.
```

Comment:
Yes, the code running on the VM can access the Genomics API. The primary purpose the Pipelines API serves today is to enable people with code that uses traditional file-based data access (BAMs and VCFs) to run those at scale.

Code that uses the GA4GH API for a "pipeline" at scale already has Dataflow available, but there will probably be use-cases where either Dataflow is a big hammer or people want to use a language not supported by Dataflow.